src/main/cpp/win.cpp

```diff
@@ -1133,4 +1133,111 @@ Java_net_rubygrapefruit_platform_internal_jni_WindowsRegistryFunctions_getValueN
     return true;
 }
 
+// File watching
```

Comment:
Move to a separate file, this one is already 1k+ lines?

Reply:
Doing this now.